### ISSUE TITLE
Adding cities validation to checkoutShippingAddressUpdateV2

### ISSUE THREAD

**Issue by itsdarrylnorris** [Shopify/js-buy-sdk#747]
### Feature details 

**Is your feature request related to a problem? Please describe.**
I have notice that the `checkoutShippingAddressUpdateV2` method only validates country, zip and province and does not take in account of cities, address1, and address2.

**Describe the solution you'd like**
It will be nice if the `checkoutShippingAddressUpdateV2` method can throw some errors in the case of city that is not correct.

**Comment by christerwpersson** [comment]
I'm fairly sure this feature request should be directed towards the Storefront api, rather than the sdk in this repo.

**Comment by itsdarrylnorris** [comment]
@christerwpersson ,

You are correct, but if I am not wrong we Shopify uses some of these issues as features request in their roadmap.
